source/manual/ruby.html.md

```diff
@@ -32,17 +32,33 @@ We set up rbenv differently depending on what's going on:
 
 ## Add a new Ruby version in puppet
 
-Build a package using [fpm](debian-packaging.html#fpm) and make it
-available using our aptly repository.
+You will need to build a new [fpm](debian-packaging.html#fpm) package with the new Ruby version.
+This package can then be copied to Aptly machine, and the new version added to puppet.
 
+### Building the fpm package.
+
+- Add a new recipe for the ruby version in [Packager](packager).
+The folder name will be the Ruby version, and contain a `recipe.rb` file. See previous entries for examples.
+The recipe will require the SHA256 of the version's `tar.gz` file.
+- Once the Packager change is merged, [build the package](jenkins).
+The Jenkins job will produce a `.deb` package in the `Last Successful Artifacts`.
```

Comment:
I think the order is not quite right.
We are saying to test this in the vm prior to merging our branch into packager but then we also say to run the jenkins job after packager is merged.
The problem with this is that the package that we want to test in the vm is the result of the jenkins job.

Reply:
thanks Alan